routers/rooms.py

```diff
@@ -77,10 +77,10 @@ async def delete_room(id: int):
         Returns:
                 Dict with result
     """
-    return await rooms_utils.delete_room(id=id)
```

Comment:
if you have delete room functionality, what will happen to all the bookings when you delete the room?

Reply:
As now it will delete too, I can remove cascade delete and remain it.

Reply:
What about the case when you delete a room for some reason. And you delete all the bookings related to it. This can cause some problems: you delete historical data about bookings(usually such data is stored for some time, for statistics  reports etc), you can delete future booking (in real life if you delete future bookings this is an issue, you'd rather keep the booking and mark it as smth that needs clarification)

Reply:
Ok, will change it to "SET NULL"